tests_zemu/tests/adr0014.test.ts

```diff
@@ -609,4 +724,134 @@ describe("Standard-Adr0014", function () {
       await sim.close();
     }
   });
+
+test.concurrent.each(models)('contracts-empty-data', async function (m) {
+    const sim = new Zemu(m.path)
+    try {
+    await sim.start({ ...defaultOptions, model: m.name });
+    const app = new OasisApp(sim.getTransport());
+
+    // Change to expert mode so we can skip fields
+    await sim.clickRight();
+    await sim.clickBoth();
+    await sim.clickLeft();
+
+    const meta = Buffer.from(
+      "ompydW50aW1lX2lkeEAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDBlMmVhYTk5ZmMwMDhmODdmbWNoYWluX2NvbnRleHR4QGIxMWIzNjllMGRhNWJiMjMwYjIyMDEyN2Y1ZTdiMjQyZDM4NWVmOGM2ZjU0OTA2MjQzZjMwYWY2M2M4MTU1MzU=",
+      "base64"
+    );
+
+    const txBlob = Buffer.from(
+      "o2F2AWJhaaJic2mBomVub25jZQBsYWRkcmVzc19zcGVjoWlzaWduYXR1cmWhZ2VkMjU1MTlYIDXD8zVt2FNk/roDVLVFraEJ0b2zi/XWEmgX24xyz9aRY2ZlZaFmYW1vdW50gkBAZGNhbGyiZGJvZHmkZGRhdGFBoGZ0b2tlbnODgkQ7msoAQIJCB9BEV0JUQ4JDLcbARFdFVEhnY29kZV9pZABvdXBncmFkZXNfcG9saWN5oWhldmVyeW9uZaBmbWV0aG9kdWNvbnRyYWN0cy5JbnN0YW50aWF0ZQ==",
+      "base64"
+    );
+
+    const path = "m/44'/474'/0'";
+    const signatureRequest = app.signRtEd25519(path, meta, txBlob)
+
+      // Wait until we are not in the main menu
+      await sim.waitUntilScreenIsNot(sim.getMainMenuSnapshot())
+
+    await sim.compareSnapshotsAndApprove(
+      ".",
+      `${m.prefix.toLowerCase()}-adr0014-sign_ed15519_contracts-empty-data`
+    );
+
+      const signatureResponse = await signatureRequest
+      console.log(signatureResponse)
+
+      expect(signatureResponse.return_code).toEqual(0x9000)
+      expect(signatureResponse.error_message).toEqual('No errors')
+
```

Comment:
Can we verify the signature here too using 'addon.schnorrkel_verify' as above? It also applies for the following tests